ToNote/Controls/NotePanel.cs

```diff
@@ -146,6 +170,24 @@ private void ConfigureRichTextBoxEvents(ExtendedRichTextBox rtb)
 
                 Note?.DeleteFile(rtb.CurrentFile);
             };
+
+            rtb.GotKeyboardFocus += (s, e) =>
+            {
+                lastFocused = (ExtendedRichTextBox)s;
+            };
+        }
+
+        /// <summary>
+        /// Applies appropriate events to the provided TodoControl
+        /// </summary>
+        /// <param name="todoControl"></param>
+        private void ConfigureTodoControlEvents(TodoControl todoControl)
+        {
+            todoControl.BackspacePressedWhileEmpty += (s, e) =>
```

Comment:
Should behave the same way as deleting an empty text box does (should switch to an item above the deleted item)